src/probnum/filtsmooth/__init__.py

```diff
@@ -1,42 +1,20 @@
-"""
-Bayesian Filtering and Smoothing.
-"""
+"""Bayesian Filtering and Smoothing."""
 
 from .bayesfiltsmooth import *
-from .filtsmoothposterior import FiltSmoothPosterior
+from .filtsmoothposterior import *
 from .gaussfiltsmooth import *
-from .statespace import *
 
 # Public classes and functions. Order is reflected in documentation.
 __all__ = [
     "Kalman",
-    "IteratedKalman",
     "ContinuousEKFComponent",
-    "ContinuousUKFComponent",
     "DiscreteEKFComponent",
+    "ContinuousUKFComponent",
     "DiscreteUKFComponent",
     "UnscentedTransform",
-    "Transition",
-    "SDE",
-    "LinearSDE",
-    "LTISDE",
-    "DiscreteGaussian",
-    "DiscreteLinearGaussian",
-    "DiscreteLTIGaussian",
     "FiltSmoothPosterior",
```

Comment:
this __init__ blew up, therfore I extracted `statespace` into its own module. Its' accessible via `probnum.filtsmooth.statespace`.

Reply:
I dont know... it was getting too big though, which is why I decided to split it up. As far as I understand, much of the statespace module will soon move to some different random-process-related module anyway?

Reply:
Shall we leave it here then or move it into toplevel with its own package?

Reply:
Ah one thing I still wondered about this: Is statespace only imported by filtsmooth, or also in other parts of the code?

I'm fine with leaving it here and moving it later, but if you think it's better to move it now feel free.

Reply:
By diffeq, too, because there are the `Integrator` priors now.

Reply:
Lets leave it for the sake of a faster review and if I change my mind, this will be a quick PR in the future